blog/2023-07-07-hidden-costs-ofmonitoring/index.md

```diff
@@ -0,0 +1,227 @@
+---
+slug: hidden-costs-of-monitoring
+title: "The Hidden Costs of Monitoring"
+authors: satya
+tags: [netdata-monitoring, costs-of-monitoring,hidden-costs,data-costs,opportunity-costs,retention-costs,cost-of-delayed-rca]
+keywords: [netdata-monitoring, costs-of-monitoring,hidden-costs,data-costs,opportunity-costs,retention-costs,cost-of-delayed-rca]
+image: ./img/hidden-costs.png
+---
+
+![stacked-netdata](./img/netdata-parents.png)
+
+When it comes to monitoring IT infrastructure, the costs you see on the price tag of the tool are often just the tip of the iceberg. Below the waterline, a mass of hidden costs can lurk, which can significantly affect the total cost of ownership. Here are some of the key areas where these hidden costs can occur.
+
+
```

Comment:
Should we have another level of titles here to differentiate commercial centralized observability solutions vs open source observability solutions?

Reply:
not sure we need another level since we only have one at each
* Prometheus & Grafana --> Open Source Observability solutions
* Datadog, Dynatrace, NewRelic and other commercial SaaS --> commercial centralized observability

maybe we could change the titles of these to something like:
* Prometheus & Grafana (Open Source references)
* Datadog, Dynatrace, NewRelic (Commercial Centralized references)